### PR TITLE
Add details

### PR DESCRIPTION
**Pull request by Krista-Rogers**
Tested by pointing sprout at this version for its submodule of sipp, and running `make full_test`, which passed.

### CODE REVIEW

File: auth.c [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2012 to the present time.
+*/
```
Comment (tomwatson1024) [inline auth.c]: This should probably be indented by a space. It's like the most minor thing ever, but it's irrationally annoying when the `*`s aren't in line 😜

File: auth.c [hunk 2]
```diff
@@ -1,3 +1,8 @@
+/**
```
Comment (tomwatson1024) [inline auth.c]: One `*` here would be consistent with the rest of the file.

File: README.txt [hunk 3]
```diff
@@ -51,7 +54,7 @@ make changes in Sipp, *PLEASE* follow a few coding rules:
     (better portability). (e.g. prefer "printf" to "cout <<" ).
 
   - Check your changes can be compiled on:
- 
+
```
Comment (tomwatson1024) [inline README.txt]: If we hadn't changed the files before, we certainly have now! Although I suppose it depends on the legal ramifications of whitespace, something I never thought I'd need to consider.

File: call.cpp [hunk 4]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2012 to the present time.
+*/
```
Comment (tomwatson1024) [inline call.cpp]: space + double `*`s as above

File: call.hpp [hunk 5]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2012 to the present time.
+*/
```
Comment (tomwatson1024) [inline call.hpp]: space + `*`

File: deadcall.cpp [hunk 6]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2012 to the present time.
+*/
```
Comment (tomwatson1024) [inline deadcall.cpp]: space + `*`

File: md5.c [hunk 7]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2012 to the present time.
+*/
```
Comment (tomwatson1024) [inline md5.c]: I'm going to go ahead and stop commenting on these, I'm sure you get the idea

File: message.cpp [hunk 8]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from December 2012 to the present time.
+*/
+
```
Comment (tomwatson1024) [inline message.cpp]: Would it make sense for this to be part of the larger block comment below?

**Review by tomwatson1024 — APPROVED** [review]
Generally looks fine, a couple of minor style comments inline.

File: message.cpp (hunk 8 above)
Comment (KristaHyer) [reply message.cpp]: I intend to leave this, so that all files have a consistent message at the top

Comment (tomwatson1024) [reply message.cpp]: Ok, makes sense 😺

**Pull request merged**